components/Audience/Audience.scss

```diff
@@ -0,0 +1,157 @@
+.audience {
+  padding: 2rem 0;
+
+  &__title {
+    margin-bottom: 0.8rem;
+
+    @include desktop {
+      margin-bottom: 1rem;
+    }
+  }
+
+  &__description {
+    margin-bottom: 1.5rem;
+    color: $color-gray;
+
+    @include tablet {
+      margin-bottom: 2rem;
+    }
+
+    @include desktop {
+      margin-bottom: 3.5rem;
+    }
+  }
+
+  &__cards {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+  }
+
+  &__card {
+    margin-bottom: 2.4%;
+    border-radius: 2rem;
+    padding: 1.3rem 1rem;
+    width: 48.7%;
+    max-height: 13.5rem;
+    background-size: cover;
+    background-repeat: no-repeat;
+    aspect-ratio: 1/2;
+
+    &:nth-child(1) {
+      background-image: url("../../public/images/audience-coffie.jpg");
+      background-position: left center;
+      background-size: 130%;
+    }
+
+    &:nth-child(2) {
+      background-image: url("../../public/images/audience-cakes.jpg");
+      background-position: left center;
+      background-size: 100%;
+
+      @include desktop {
+        background-position: -1.9rem center;
```

Comment:
Such meticulous adaptation of the position of every single image is in 90% of cases is unnecessary, we would just be satisfied with "background-position: left center" in most cases, it would be good enough. 
Sometimes it can be required, for example if there's a person's face in the picture and you want it to be centered, or you want some element to be fully visible.
But here it is an overhead, so I suggest "left center" only